bucket/lockhunter.json

```diff
@@ -3,8 +3,17 @@
     "description": "Delete files blocked by something you do not know.",
     "homepage": "https://lockhunter.com/index.htm",
     "license": "Unknown",
-    "url": "https://lockhunter.com/assets/exe/lockhuntersetup_3-4-3.exe",
-    "hash": "2c0d52dca3e5ce9cfc8062cb72c7235e3c6ff650242fcb7d46a892d602dd3dd1",
+    "depends": "sudo",
+    "url": [
+        "https://lockhunter.com/assets/exe/lockhuntersetup_3-4-3.exe",
+        "https://raw.githubusercontent.com/phanirithvij2000/Extras/lockhunter-shellex/scripts/lockhunter/install-context.reg",
```

Comment:
Will change this to `ScoopInstaller/Extras/master` once I get the script working locally

Reply:
Why not add them as https://github.com/ScoopInstaller/Extras/tree/master/scripts/lockhunter/install-context.reg and
https://github.com/ScoopInstaller/Extras/tree/master/scripts/lockhunter/uninstall-context.reg

and them change
`$dir\\install-context.reg`
to
`$scoopdir/apps/scoop/current/scripts/lockhunter/install-context.reg`
per https://github.com/ScoopInstaller/Scoop/wiki/Pre--and-Post-install-scripts#variables

That makes more sense them unnecessarily downloading files that are already in the repo.